### PR TITLE
Getting started documentation

### PR DESCRIPTION
**Pull request by ThomasRitense**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -10,10 +10,10 @@
...
-  * [First dive into Valtimo](valtimo-implementation/first-dive/first-dive.md)
-    * [Docker container](valtimo-implementation/first-dive/docker-container/docker-container.md)
-    * [Creating your own Valtimo implementation](valtimo-implementation/first-dive/create-your-own/create-your-own.md)
+* [Getting started](getting-started/getting-started.md)
+  * [First dive into Valtimo](getting-started/first-dive/first-dive.md)
+    * [Docker container](getting-started/first-dive/docker-containers/docker-container.md)
+    * [Creating your own Valtimo implementation](getting-started/first-dive/create-your-own/create-your-own.md)
   * [Compatibility matrix](valtimo-implementation/compatibility-matrix.md)
   * [Database support](valtimo-implementation/database-support.md)
   * [Modules](valtimo-implementation/modules/modules.md)
```
Comment (marijnritense) [inline SUMMARY.md:19]: Pretty much all links here no longer work. Can you fix this?

File: getting-started/starting-up-valtimo/start-as-applications/start-as-applications.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Start Valtimo as applications
```
Comment (marijnritense) [inline getting-started/starting-up-valtimo/start-as-applications/start-as-applications.md:1]: As applications?

File: getting-started/starting-up-valtimo/starting-up-valtimo.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Valtimo consists of a frontend (Angular) and a backend (Java/Kotlin) application. Both services have their own Github repository with instructions on how the service can be started.
+
+## Start up strategies
+The Valtimo application can be started in two ways:
+* As Docker containers
+* As applications
+
+When starting the Valtimo applications as Docker containers, all supporting services and the two Valtimo services are started up with a Docker Compose file. Instructions on how to start Valtimo as Docker containers can be found [here](valtimo-documentation/getting-started/starting-up-valtimo/start-as-docker-containers/run-as-docker-containers.md).
+
+When starting the Valtimo applications as applications, the supporting services are started up with a Docker Compose file. The two Vatlimo services are started up from their respective Git repositories using a command line interface (CLI) or integrated development environment
```
Comment (marijnritense) [inline getting-started/starting-up-valtimo/starting-up-valtimo.md:21]: Valitmo -> Valtimo

File: extending-valtimo/plugin/custom-plugin-definition.md [hunk 4]
```diff
@@ -571,7 +571,7 @@ translation.
 ### Plugin module
 
 Finally, after implementing the components and specification, a module has to be defined for the plugin.
-This module, together with the specification, is then imported in the app module as shown in [on this page](../../valtimo-implementation/modules/core/plugin.md).
+This module, together with the specification, is then imported in the app module as shown in [on this page](valtimo-documentation/getting-started/modules/core/plugin.md).
```
Comment (ivo-ritense) [inline extending-valtimo/plugin/custom-plugin-definition.md:574]: i don't think this link will work as the `valtimo-documentation` part is a reference to the repo itself. To make a path from the root of the documentation i believe you can just do `/getting-started/...`

**Review by ivo-ritense — APPROVED** [review]

**Review by ivo-ritense — APPROVED** [review]

**Pull request merged**